strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/CleanupExpiredArtifactsFromProxyRepositoriesCronJobTestIT.java

```diff
@@ -78,7 +83,9 @@ public void expiredArtifactsCleanupCronJobShouldCleanupDatabaseAndStorage()
                                                                                              path);
         assertThat(artifactEntryOptional, CoreMatchers.equalTo(Optional.empty()));
 
-        try (final InputStream ignored = proxyRepositoryProvider.getInputStream(storageId, repositoryId, path))
+        Path repositoryPath = proxyRepositoryProvider.fetchPath(repositoryPathResolver.resolve(storageId, repositoryId,
```

Comment:
@sbespalov : Just for my better understanding, what does `fetchPath` do exactly?

Reply:
I made some JavaDocs in `RepositoryProvider` interface to make it clearer.